modules/ROOT/pages/apikit-graphql-api-mapping.adoc

```diff
@@ -0,0 +1,62 @@
+= Mapping a GraphQL API to Your Data Sources
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+include::reuse::partial$beta-banner.adoc[tag="apikit"]
+
+== Select Sources
+
+The module sources vary depending on the implementation of your GraphQL API. APIkit for GraphQL provides two kind of sources:
+
+* Data Fetchers
+* Data Loaders
+
+Your selection depends on your implementation needs. 
+
+GraphQL is transport-layer agnostic, so you choose the main connector listener depending on the protocol that you want to serve your data.
+
+=== Add a Listener
+
+The inbound endpoint that triggers the main application flow is the connector listener specific to the client-server protocol of your choice. It is common that GraphQL communicates over HTTP, so APIkit scaffolder generates an HTTP listener by default, but you can choose another source for serving your data over a different protocol.
```

Comment:
Avoid "it is" construction.

GraphQL often communicates over....

Reply:
Sure! Thanks!